client/src/components/Rating.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { ModuleWrapper, RatingBureau, RatingContainer, RatingDescriptors, RatingInfo } from './styles/styles';
+
+const Rating = ({ rating }) => (
+  <ModuleWrapper>
+    <RatingContainer>
+      <RatingInfo>
+        <div className="game_rating_icon">
+          <img alt="mature rating" src="https://store.akamai.steamstatic.com/public/shared/images/game_ratings/ESRB/m.png" />
```

Comment:
Solid component - was wondering if this mature game image would ever change? Like would it ever be T for teen or E for everyone? If so, may want to find a way to display other images depending on the rating?

Reply:
Yep, on the agenda. Just trying to get a MVP, and I have that and 3 other things on the list. I want to have a whole lot more dynamism and pop.